.github/workflows/test.yml

```diff
@@ -52,6 +52,13 @@ jobs:
       run: |
         sudo apt-get update
         sudo apt-get install -y libpcap-dev
+    - name: Install Cassandra CPP driver
+      run: |
+        wget https://downloads.datastax.com/cpp-driver//ubuntu/18.04/cassandra/v2.16.0/cassandra-cpp-driver_2.16.0-1_amd64.deb
```

Comment:
We can use the following to background them all and wait for completion instead of doing it sequentially:

```bash
wget https://downloads.datastax.com/cpp-driver//ubuntu/18.04/cassandra/v2.16.0/cassandra-cpp-driver_2.16.0-1_amd64.deb & 
wget https://downloads.datastax.com/cpp-driver/ubuntu/18.04/cassandra/v2.16.0/cassandra-cpp-driver-dev_2.16.0-1_amd64.deb & 
wget https://downloads.datastax.com/cpp-driver/ubuntu/18.04/dependencies/libuv/v1.35.0/libuv1_1.35.0-1_amd64.deb & 
wget https://downloads.datastax.com/cpp-driver/ubuntu/18.04/dependencies/libuv/v1.35.0/libuv1-dev_1.35.0-1_amd64.deb &
wait
```